index.html

```diff
@@ -41,442 +19,942 @@
+                  <tr>
+                    <td class="table-borderless">Founders and Advisors</td>
+                    <td class="table-borderless">14%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Marketing and R$D</td>
+                    <td class="table-borderless">25%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Reserve</td>
+                    <td class="table-borderless">10%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Network</td>
+                    <td class="table-borderless">10%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Operation</td>
+                    <td class="table-borderless">10%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Liquidity</td>
+                    <td class="table-borderless">20%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Staking</td>
+                    <td class="table-borderless">10%</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Community and Daopad sales</td>
+                    <td class="table-borderless">1%</td>
+                  </tr>
+                </table>
+              </div>
+              <div class="col-lg-4 content mb-2">
+                <!-- <h2>TOKEN SALE</h2>-->
+                <h3>Token Stats</h3>
+
+                <table class="table table-responsive">
+                  <tr>
+                    <td class="table-borderless">Total Token Supply</td>
+                    <td class="table-borderless">2.000.000.000</td>
+                  </tr>
+
+                  <tr>
+                    <td class="table-borderless">Total Token locked</td>
+                    <td class="table-borderless">$0.20/token</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Token Price</td>
+                    <td class="table-borderless">2.000.000.000</td>
+                  </tr>
+
+                  <tr>
+                    <td class="table-borderless">Soft Cap</td>
+                    <td class="table-borderless">$1.000.000</td>
+                  </tr>
+                  <tr>
+                    <td class="table-borderless">Hard Cap</td>
+                    <td class="table-borderless">$2.000.000</td>
+                  </tr>
+                </table>
+              </div>
+            </div>
+            <div class="row mt-3">
+              <div class="col-lg-4 about-img">
+                <h2>ROADMAP</h2>
+              </div>
+
+              <div class="col-lg-4 content">
+                <ul>
+                  <li>
+                    <i class="ion-android-checkmark-circle"></i>Project
+                    Community 2021
```

Comment:
![image](https://user-images.githubusercontent.com/60205816/179096459-43168eb9-7105-4487-97df-5318cefcf66e.png)

Kindly add a margin or padding between the border and text on mobile screen

Reply:
added margin. thanks for the feedback